### ISSUE TITLE
leading backslash

### ISSUE THREAD

**Issue by dicktyr** [rockandska/fzf-obc#27]
I noticed this as I sometimes use backslash to override aliases

```
$ \cd <tab>
$ \cd ^[\[01\;34m/usr/^[\[0m
```
note that `mv \x <tab>` works as expected
so it seems the issue is when backslash is the first non-blank character

**Comment by rockandska** [comment]
Thanks for your feedback.
I'm able to reproduce it too and will take a look soon.

NB: it seems that the complete function used when adding \ to bypass the alias do not use the original completion script. So it could lead to unexpected results as see files when doing \cd
If you test in a clean environment (without fzfz-obc) you'll see:
```
complete -F _minimal \cd
complete -o nospace -F _cd cd
```

**Comment by rockandska** [comment]
Please, test with branch fix/#27 to let me know if it solved your problem before merging it.
Thanks

**Comment by dicktyr** [comment]
thanks
works for me
